includes/controls/settings-footer.php

```diff
@@ -0,0 +1,535 @@
+				'label_on' => 'Show',
+				'label_off' => 'Hide',
+				'selector' => '.site-footer .site-description',
+			]
+		);
+
+		$this->add_control(
+			'hello_footer_menu_display',
+			[
+				'type' => Controls_Manager::SWITCHER,
+				'label' => __( 'Menu', 'hello-elementor' ),
+				'default' => ( false === get_option( 'hello_footer_type_default' ) ? 'yes' : false ),
+				'label_on' => 'Show',
+				'label_off' => 'Hide',
+				'selector' => '.site-footer .site-navigation',
+			]
+		);
+
+		$this->add_control(
+			'hello_footer_copyright_display',
+			[
+				'type' => Controls_Manager::SWITCHER,
+				'label' => __( 'Copyright', 'hello-elementor' ),
+				'default' => ( false === get_option( 'hello_footer_type_default' ) ? 'yes' : false ),
+				'label_on' => 'Show',
+				'label_off' => 'Hide',
+				'selector' => '.site-footer .copyright',
+			]
+		);
+
+		$this->add_control(
+			'hello_footer_layout',
+			[
+				'type' => Controls_Manager::SELECT,
+				'label' => __( 'Layout', 'hello-elementor' ),
+				'options' => [
+					'default' => __( 'Default', 'hello-elementor' ),
+					'inverted' => __( 'Inverted', 'hello-elementor' ),
+					'stacked' => __( 'Centered', 'hello-elementor' ),
+				],
+				'selector' => '.site-footer',
+				'default' => 'default',
+			]
+		);
+
+		$this->add_control(
+			'hello_footer_width',
+			[
+				'type' => Controls_Manager::SELECT,
+				'label' => __( 'Width', 'hello-elementor' ),
+				'options' => [
+					'boxed' => __( 'Boxed', 'hello-elementor' ),
+					'full-width' => __( 'Full Width', 'hello-elementor' ),
+				],
+				'selector' => '.site-footer',
+				'default' => 'boxed',
+			]
+		);
+
+		$this->add_responsive_control(
+			'hello_footer_custom_width',
+			[
+				'type' => Controls_Manager::SLIDER,
+				'label' => __( 'Content Width', 'hello-elementor' ),
+				'size_units' => [
+					'%',
+					'px',
+				],
+				'default' => [
+					'size' => '1140',
+				],
+				'range' => [
+					'px' => [
+						'max' => 2000,
+						'step' => 1,
+					],
+					'%' => [
+						'max' => 100,
+						'step' => 1,
+					],
+				],
+				'condition' => [
+					'hello_footer_width' =>'boxed',
+				],
+				'selectors' => [
+					'.site-footer' => 'width: {{SIZE}}{{UNIT}}; max-width: 100%;',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'hello_footer_gap',
+			[
+				'type' => Controls_Manager::SLIDER,
+				'label' => __( 'Gap', 'hello-elementor' ),
+				'size_units' => [
+					'%',
+					'px',
+				],
+				'default' => [
+					'size' => '0',
+				],
+				'range' => [
+					'px' => [
+						'max' => 2000,
+						'step' => 1,
+					],
+					'%' => [
+						'max' => 100,
+						'step' => 1,
+					],
+				],
+				'selectors' => [
+					'.site-footer' => 'padding-right: {{SIZE}}{{UNIT}}; padding-left: {{SIZE}}{{UNIT}}',
+				],
+				'condition' => [
+					'hello_footer_layout!' => 'stacked',
+				],
+			]
+		);
+
+		$this->add_group_control(
+			\Elementor\Group_Control_Background::get_type(),
```

Comment:
?

Reply:
If you have a use statement, you don't need the full qualifier.

Reply:
Fixed.